build.gradle

```diff
@@ -34,7 +34,8 @@ dependencies {
 
 tasks.withType(ScalaCompile) {
     scalaCompileOptions.additionalParameters = [
-            "-language:postfixOps", "-Xfatal-warnings"
+            "-language:postfixOps", "-Xfatal-warnings",
+            "-Ypatmat-exhaust-depth", "off" // TODO: Remove once the "stupid mapping" in Simplifier.scala is removed
```

Comment:
- Remove comment.
- one option per line.

Reply:
I'm not sure what you mean by one option per line? `-Ypatmat-exhaust-depth` is a single option.

Reply:
There are 3 options and you have them on 2 lines

Reply:
I want to keep the flag, so you can remove the comment! :)